client/azure_client_set.go

```diff
@@ -209,7 +209,7 @@ func newPublicIPAddressesClient(authorizer autorest.Authorizer, subscriptionID,
 	return &client, nil
 }
 
-func newSecurityGroupsClient(authorizer autorest.Authorizer, subscriptionID, partnerID string) (*network.SecurityRulesClient, error) {
+func newSecurityRulesClient(authorizer autorest.Authorizer, subscriptionID, partnerID string) (interface{}, error) {
```

Comment:
Could we keep both `SecurityGroupsClient` and `SecurityRulesClient`?

Reply:
It was a typo. The method was called `getSecurityGroupsClient` but was returning a `SecurityRulesClient`